Marlin/src/lcd/menu/menu_tramming.cpp

```diff
@@ -94,6 +98,10 @@ void goto_tramming_wizard() {
   set_all_unhomed();
   queue.inject_P(TERN(CAN_SET_LEVELING_AFTER_G28, PSTR("G28L0"), G28_STR));
 
+  // Initialize points to invalid so we will know if they have been measured.
+  LOOP_L_N(i, G35_PROBE_COUNT) z_isvalid[i] = false;
```

Comment:
`ZERO(z_isvalid)` would also be fine here, but I'm not sure whether it results in smaller code. Probably the compiler knows to convert `memset` into a small loop in some cases.

Reply:
As mentioned above, there doesn't seem to be any place where `z_isvalid[...]` is being set to `true`. Has this new code actually been tested?